File: matiasmasso/Mat.NET/DT2/DTOAuditStock.vb
Public Class DTOAuditStock
    Inherits DTOBaseGuid

    Property Year As Integer
    Property Exercici As DTOExercici
    Property Ref As String
    Property Sku As DTOProductSku
    Property Dsc As String
    Property Qty As Integer
    Property Palet As String
    Property FchEntrada As Date
    Property Dias As Integer
    Property Entrada As String
    Property Procedencia As String
    Property Cost As Decimal

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOAuditStock.cs
using System;

namespace DTO
{
    public class DTOAuditStock : DTOBaseGuid
    {
        public int Year { get; set; }
        public DTOExercici Exercici { get; set; }
        public string Ref { get; set; }
        public DTOProductSku Sku { get; set; }
        public string Dsc { get; set; }
        public int Qty { get; set; }
        public string Palet { get; set; }
        public DateTime FchEntrada { get; set; }
        public int Dias { get; set; }
        public string Entrada { get; set; }
        public string Procedencia { get; set; }
        public decimal Cost { get; set; }

        public DTOAuditStock() : base()
        {
        }

        public DTOAuditStock(Guid oGuid) : base(oGuid)
        {
        }
    }
}
